doc/user-guide/plotting-2d.rst

```diff
@@ -0,0 +1,323 @@
+.. currentmodule:: xarray
+.. _plotting.2d:
+
+2D Plots
+========
+
+The default method :py:meth:`DataArray.plot` calls :py:func:`xarray.plot.pcolormesh`
+by default when the data is two-dimensional.
+
+
+.. jupyter-execute::
+
+    import matplotlib.pyplot as plt
+    import numpy as np
+    import pandas as pd
+    import xarray as xr
+
+    # Load example data
+    airtemps = xr.tutorial.open_dataset("air_temperature")
+    air = airtemps.air - 273.15
+    air.attrs = airtemps.air.attrs
+    air.attrs["units"] = "deg C"
+
+    air2d = air.isel(time=500)
+    air2d.plot();
+
+All 2d plots in xarray allow the use of the keyword arguments ``yincrease``
+and ``xincrease``.
+
+.. jupyter-execute::
+
+    air2d.plot(yincrease=False);
+
+.. note::
+
+    We use :py:func:`xarray.plot.pcolormesh` as the default two-dimensional plot
+    method because it is more flexible than :py:func:`xarray.plot.imshow`.
+    However, for large arrays, ``imshow`` can be much faster than ``pcolormesh``.
+    If speed is important to you and you are plotting a regular mesh, consider
+    using ``imshow``.
+
+Coordinate Handling
+-------------------
+
+If you'd like to find out what's really going on in the coordinate system,
+read on.
+
+.. jupyter-execute::
+
+    import cartopy.crs as ccrs
```

Comment:
please move this import up to the first code cell, and remove the second cartopy import further down in this page.